x-pack/test/security_solution_endpoint/apps/endpoint/endpoint_list.ts

```diff
@@ -79,7 +79,7 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
         await testSubjects.exists('emptyPolicyTable');
       });
 
-      it('finds data after load and polling', async () => {
+      it.skip('finds data after load and polling', async () => {
```

Comment:
We can rework this test.  For polling, it would probably make sense to load data directly in the destination index to ensure that we update the UI correctly.  We could break it out to another, isolated integration test where we can load data to the destination index to test this particular case.  Alternatively, we get the transform to run more quickly for the test, but that sounds non-trivial.

Reply:
@kevinlog To cut down cost I suggest that we move this to when data is loaded. I believe you were trying to test the timer refresh?